### PR TITLE
quick fixes, non-anniversary

### PR DESCRIPTION
**Pull request by syrifgit**
spreist MF:I
resto sham old pack string (fixes healing helper)

### CODE REVIEW

File: TheWarWithin/Priorities/PriestShadow.simc [hunk 1]
```diff
@@ -138,7 +138,7 @@ actions.filler+=/shadow_word_pain,cycle_targets=1,if=set_bonus.tier31_4pc
 actions.filler+=/shadow_word_pain,cycle_targets=1,if=!set_bonus.tier31_4pc
 
 actions.empowered_filler+=/mind_spike_insanity,cycle_targets=1
-actions.empowered_filler+=/mind_flay_insanity,cycle_targets=1
+actions.empowered_filler+=/mind_flay_insanity,cycle_targets=1,,if=buff.mind_flay_insanity.up
```
Comment (Hekili) [inline TheWarWithin/Priorities/PriestShadow.simc]: 1 comma.

File: TheWarWithin/Priorities/PriestShadow.simc [hunk 2]
```diff
@@ -138,7 +138,7 @@ actions.filler+=/shadow_word_pain,cycle_targets=1,if=set_bonus.tier31_4pc
 actions.filler+=/shadow_word_pain,cycle_targets=1,if=!set_bonus.tier31_4pc
 
 actions.empowered_filler+=/mind_spike_insanity,cycle_targets=1
```
Comment (Hekili) [inline TheWarWithin/Priorities/PriestShadow.simc]: Please add `,if=buff.mind_spike_insanity.up` (for the same reason the Mind Flay change is needed below.

**Review by Hekili — CHANGES_REQUESTED** [review]
Quick fixes to PriestShadow; please import if can.

**Comment by syrifgit** [comment]
Done. Servers are down, will need to make a list of pack strings to generate once servers are up for anything we do from this point on.

Unless we can convince someone on EU to do it.

**Review by Hekili — APPROVED** [review]

**Pull request merged**